<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinePlotOptionsClass</class>
 <widget class="QWidget" name="LinePlotOptionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinePlotOptions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetLogOptions" native="true">
     <layout class="QHBoxLayout" name="widgetLog">
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="plotYLabel">
        <property name="text">
         <string>Plot Y axis:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckLog10">
        <property name="text">
         <string>Log10 </string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetPlotAxis" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Plot X axis:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>12</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetNorm" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Normalization:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radNoNormalization">
        <property name="toolTip">
         <string>No normalization: display the raw signal along the line</string>
        </property>
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radVolumeNormalization">
        <property name="toolTip">
         <string>Normalize by the volume of the bin/box at each point</string>
        </property>
        <property name="text">
         <string>Volume</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radNumEventsNormalization">
        <property name="toolTip">
         <string>Normalize signal by the number of events contributing to it</string>
        </property>
        <property name="text">
         <string>Num. Events</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>23</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
